st2common/st2common/content/loader.py

```diff
@@ -204,6 +240,36 @@ def _get_folder(self, pack_dir, content_type):
             return None
         return path
 
+    def _get_runners_from_dir(self, base_dir):
+        result = {}
+        for runner_name in os.listdir(base_dir):
```

Comment:
Perhaps we should sort the `os.listdir` result to always register runners in the same order? Or maybe it's not a big deal.